### DISCUSSION TITLE
make_future_dataframe with "freq" parameter

### DISCUSSION THREAD

**Discussion by mixcheck** [ourownstory/neural_prophet#1383] [Q&A - get help using NeuralProphet]
Hi all,

thank you for your development time-series python library.

I want to forecast with freq=MS.

I make neuralprophet model below. My data is monthly frequency data (2020-01-01, 2020-02-01 ... )

m.fit(all_data, epochs=1000, **freq='MS'**)
future = m.make_future_dataframe(all_data, periods=730) # or future = m.make_future_dataframe(all_data, periods=24) 
forecast = m.predict(future)

but when I use make_future_dataframe, It does not forecast 2 years. 

plz help me...

**Comment by SimonWittner** [comment]
Hi, 
how does your all_data and outcome look like? 
Did you maybe enable autoregression by defining the "n_lags" parameter in the model definition?

**Comment by leoniewgnr** [comment]
@mixcheck do you still have that problem?
